File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/c/cwe-79/code_before3.c
// Source: Row 17 in ./dataset/CVEfixes/Analysis/results/C/df_c_cwe_79.xlsx

rndr_quote(struct buf *ob, const struct buf *text, void *opaque)
{
	if (!text || !text->size)
		return 0;

	BUFPUTSL(ob, "<q>");
	bufput(ob, text->data, text->size);
	BUFPUTSL(ob, "</q>");

	return 1;
}

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/python/cwe-79/code_before3.py
# Source: Row 159 in ./dataset/CVEfixes/Analysis/results/Python/df_python_cwe_79.xlsx

def format_errormsg(message: str) -> str:
    """Match account names in error messages and insert HTML links for them."""
    match = re.search(ACCOUNT_RE, message)
    if not match:
        return message
    account = match.group()
    url = flask.url_for("account", name=account)
    return (
        message.replace(account, f'<a href="{url}">{account}</a>')
        .replace("for '", "for ")
        .replace("': ", ": ")
    )